### DISCUSSION TITLE
Welcome to Express-Store Discussions!

### DISCUSSION THREAD

**Discussion by LesterCerioli** [LesterCerioli/Express-Store#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LesterCerioli** [comment]
I understand that it's important to implement Stock microservices 'cause we need to provide stock control features.

**Comment by karolinagusmao** [comment]
Hello there! I'm Karolina, I'm going to start working now here at the Express Store!

**Comment by ludson96** [comment]
Hey guys! I'm Ludson and I will join the Express Store team.

**Comment by giohpergentino** [comment]
Hello guys! I'm Giovanna and I will join the Express Store team.

**Comment by ReisLuizito** [comment]
Hello folks, I'm Luiz Guilherme, and I will start contributing to the Express Store team.
